codegen/src/api/mod.rs

```diff
@@ -320,10 +321,13 @@ impl RuntimeGenerator {
                     should_gen_docs,
                 )?;
 
+                let errors = errors::generate_error_type_alias(&type_gen, pallet, should_gen_docs)?;
```

Comment:
nit: We could name this something like `error_alias` to be a bit more explicit, however you prefer here